cache.go

```diff
@@ -0,0 +1,37 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+	"os"
+	"time"
+
+	cache "github.com/victorspringer/http-cache"
+	"github.com/victorspringer/http-cache/adapter/memory"
+)
+
+func cacheHandle(h http.Handler, ttl int) http.Handler {
+
+	memcached, err := memory.NewAdapter(
+		memory.AdapterWithAlgorithm(memory.LRU),
+		memory.AdapterWithCapacity(10_000_000),
+	)
+
+	if err != nil {
+		fmt.Println(err)
+		os.Exit(1)
+	}
+
+	cacheClient, err := cache.NewClient(
+		cache.ClientWithAdapter(memcached),
+		cache.ClientWithTTL(time.Duration(ttl)*time.Second),
+		cache.ClientWithRefreshKey("opn"),
```

Comment:
@anbraten What does `opn` stand for?

Reply:
@lukashass I think it can be used as query parameter to refresh the cache.